src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -282,46 +282,55 @@ def generate_FDB_2_default_config(self, filename):
     device_b = json.load(
         open(os.path.join('testcases', 'testdata', 'device_b.json')))
 
+    # DPA database test harness configuration
+    dpa_database_config = {
+        'hostName': 'localhost',
+        'port': 8000,
+        'fileUrl': '/db_sync',
+        'filePath': os.path.join('testcases', 'testdata', 'fdb_2', 'FDB_2_Portal_DPAs.kml'),
```

Comment:
I think you're working from a slightly earlier version of the file. Please use the one I forwarded yesterday in the email titled "3.5GHz: Updated NTIA kmls" as reference. (Alternatively, use the same format but completely make up the data. But that's probably more work.)

Reply:
Modified the code based on the latest  "Portal_DPAs.kml" file.